nixos/modules/security/wrappers/wrapper.c

```diff
@@ -23,182 +25,172 @@ extern char **environ;
 
 // The WRAPPER_DIR macro is supplied at compile time so that it cannot
 // be changed at runtime
-static char * wrapperDir = WRAPPER_DIR;
+static char *wrapper_dir = WRAPPER_DIR;
 
 // Wrapper debug variable name
-static char * wrapperDebug = "WRAPPER_DEBUG";
-
-// Update the capabilities of the running process to include the given
-// capability in the Ambient set.
-static void set_ambient_cap(cap_value_t cap)
-{
-    capng_get_caps_process();
-
-    if (capng_update(CAPNG_ADD, CAPNG_INHERITABLE, (unsigned long) cap))
-    {
-        perror("cannot raise the capability into the Inheritable set\n");
-        exit(1);
+static char *wrapper_debug = "WRAPPER_DEBUG";
+
+#define CAP_SETPCAP 8
+
+#if __BYTE_ORDER == __BIG_ENDIAN
```

Comment:
Unfamiliar with C best practices (or if this would even work), but maybe byteswap should be `include`d in here, rather than unconditionally?

```suggestion
#if __BYTE_ORDER == __BIG_ENDIAN
#include <byteswap.h>
```

Reply:
I think it is better to always include it so errors, so potential errors from including are catched on little endian as well. It won't have any runtime overhead in any case.